cattle/plugins/volmgr/handler.py

```diff
@@ -103,7 +103,11 @@ def _do_snapshot_backup(self, snapshot, storage_pool, progress):
                                blockstore_uuid)
         log.info("Backed up snapshot %s for volume %s" % (
             snapshot_uuid, volume_uuid))
-        snapshot.data.fields['backup'] = True
+        latest_snapshots_set = set([snapshot_uuid])
```

Comment:
That serialized lock is in the go code, right?  Is it a blocking lock?  Does it hang until it can be acquired or does it fail?